### PR TITLE
GH #11 User Expiration Date

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the permissions/add user topics to include the **Expiration Date** and related descriptor. It also updates the screen shot to include the new field.

## Affected documentation pages

- https://docs.magento.com/m2/ee/user_guide/system/permissions-users-all.html
- https://docs.magento.com/m2/ee/user_guide/stores/admin-account.html
- https://docs.magento.com/m2/ee/user_guide/system/permissions-user-roles.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional Information

Fixes gh#11

### CODE REVIEW

File: src/system/permissions-users-all.md [hunk 1]
```diff
@@ -29,39 +33,47 @@ _Example Admin Users_
...
 
-    - In the **Password Confirmation** box, repeat the password to make sure it was entered correctly.
+    - For **Password Confirmation**, re-enter the password to make sure it was entered correctly.
 
     - If your store has multiple languages, set **Interface Locale** to the language to be used for the Admin interface.
 
 1. Set **This Account is** to `Active`.
 
-1. Under _Current User Identity Verification_, enter **Your Password**.
+1. Click the calendar icon to set the **Expiration Date** for the user account.
```
Comment (leslietilling) [inline src/system/permissions-users-all.md:55]: Is the new expiration available in both Open Source and Commerce? Just want to make sure we don't need conditioning here.

File: src/system/permissions-users-all.md [hunk 2]
```diff
@@ -29,39 +33,47 @@ _Example Admin Users_
...
+
+    Defining an expiration date is helpful when a user or role is temporary. After the defined expiration date, the user account status changes to `Inactive` and can be updated, if neededß.
```
Comment (leslietilling) [inline src/system/permissions-users-all.md:57]:
```suggestion
    Defining an expiration date is helpful when a user or role is temporary. After the expiration date, the user account status changes to `Inactive` and can be updated, if needed.
```

File: src/system/permissions-user-roles.md [hunk 3]
```diff
@@ -50,16 +50,13 @@ To give someone restricted access to the Admin, the first step is to create a ro
...
 
-## Edit user role assignments
+## Assign a role to user(s)
 
 1. From the **Roles** grid, open the record in edit mode.
 
-1. Under _Current User Identity Verification_, enter **Your Password**.
-
-    ![Enter your password to edit the role]({% link images/images/permissions-role-edit.png %}){: .zoom}
-    _Editing a Role_
```
[the remark this answers is not in the document]
Comment (hickskenh) [reply src/system/permissions-user-roles.md:62]: good catch.  deleted in commit to follow

File: src/system/permissions-users-all.md (hunk 1 above)
Comment (hickskenh) [reply src/system/permissions-users-all.md:55]: was unable to find any restrictions in the ticket trails and this is community development.

**Comment by hickskenh** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Pull request merged**